src/phases/query/Multiplanner.yml

```diff
@@ -26,15 +26,15 @@ QueryKnobTemplate:
 
 InsertTemplate:
   Name: CreateDataset
-  Type: Loader
-  Threads: &threads {^Parameter: {Name: "threads", Default: {unused: "Invalid thread count."}}}
+  Type: MonotonicLoader
```

Comment:
I wonder if this might cause issues with existing tests that define an _id field. Maybe better to do this as a separate ticket for all our workloads?

Reply:
Yeah, I had originally done this to maintain consistency with other workloads that I'm changing in a different ticket, but I see your point. I'll revert this here and make the change in PERF-5269.